src/ngraph_encapsulate_clusters.cc

```diff
@@ -330,7 +330,7 @@ Status EncapsulateClusters(Graph* graph, int graph_id,
     Node* n;
     Status status = NodeBuilder(ss.str(), "NGraphEncapsulate")
                         .Attr("ngraph_cluster", cluster_idx)
-                        .Attr("_ngraph_backend", cluster_backend)
+                        .Attr("ngraph_backend", cluster_backend)
```

Comment:
Normal TF nodes are marked with `_ngraph_backend`, since underscore is needed to name an undeclared field.
Before this `NGraphEncapsulate`, our custom tf node was also using `_ngraph_backend`. In this PR, we make `ngraph_backend` a declared field, to avoid confusion.